src/rust/engine/logging/src/logger.rs

```diff
@@ -156,6 +177,22 @@ impl<W: Write + Send + 'static> Log for MaybeWriteLogger<W> {
     if !self.enabled(record.metadata()) {
       return;
     }
+    let mut should_log = self.show_rust_3rdparty_logs;
```

Comment:
I really would like this code to go in `enabled()`, but I think that `log::Metadata` does not have `module_path`. Sigh.